docs/cosmology/dev.rst

```diff
@@ -35,16 +33,15 @@ following:
 
         ... function code here ...
 
-This ensures that all code consistently uses the default cosmology
-unless explicitly overridden.
+This ensures that all code consistently uses the default cosmology unless
+explicitly overridden.
 
 .. note::
 
     In general it is better to use an explicit cosmology (for example
```

Comment:
This seems very strange after the above - the user has just been told to use a default cosmology and now that is reversed! Maybe "If you are preparing a paper and thus need to ensure your code provides reproducible results, it is better to ..."

Reply:
Yes. I've implemented your suggestion + a little more explanation.